### PR TITLE
Myriad small fixes, mostly spelling and grammar.

### PR DESCRIPTION
**Pull request by nerble**
- Spelling in some areas
- Tweaked affected-by spell display so you get grammatical strings for 1 hour and less than an hour.  Also slightly widened so the globe string fits in there now (thank me later, spellers)
- Fixed some send_to_chars that were broken (they included pronoun variables)
- Probably some other minor things I've forgotten about

### CODE REVIEW

File: server/area/three_trials.are [hunk 1]
```diff
@@ -134,7 +134,7 @@ The grotto beast stirs,  then attacks...
 ~
 This strange creature has a bulbous body which emits a cascade of
 differing shades and hues of light.  Its huge ugly head has two rows of
-wickedly razorlike teeth.  It's claws could rend adamantium and it 's tail
+wickedly razorlike teeth.  Its claws could rend adamantium and its tail
```
Comment (fromage-fraser) [inline server/area/three_trials.are:137]: The DD classic.

File: server/src/sft.c [hunk 2]
```diff
@@ -616,9 +616,9 @@ void do_web (CHAR_DATA *ch, char *argument)
         if (number_percent() < chance) 
         {
                 AFFECT_DATA af;
-                act ("You skillfully spin a sticky web around $N, trapping $m here!",
+                act ("You skillfully spin a sticky web around $N, trapping $M here!",
                      ch, NULL, victim, TO_CHAR);
-                act ("$n skillfully spins a sticky web around $N, trapping $m here!",
+                act ("$n skillfully spins a sticky web around $N, trapping $M here!",
```
Comment (fromage-fraser) [inline server/src/sft.c:621]: I think we want `$c` here: i.e. `$n` but capitalised.

Comment (fromage-fraser) [reply server/src/sft.c:621]: In fact I think we probably want to look at that usage throughout.

File: server/area/three_trials.are (hunk 1 above)
Comment (nerble) [reply server/area/three_trials.are:137]: Like homophones most spell checkers will miss it, and you have to read all the text to find them. (((0))((0)))

File: server/src/sft.c (hunk 2 above)
Comment (nerble) [reply server/src/sft.c:621]: Fair enough, though there are hundreds of uses and I'd rather push that through as a separate change after some testing to make sure it's always the behaviour we want.

**Review by shadenz1 — APPROVED** [review]
Looks good

**Review by shadenz1 — APPROVED** [review]
This'll be the merge bit

**Pull request merged**